server/controllers.js

```diff
@@ -0,0 +1,28 @@
+require( 'dotenv' ).config();
+const path = require( 'path' );
+const axios = require( 'axios' );
+
+const api = path.join( 'https://app-hrsei-api.herokuapp.com/api/fec2/', process.env.CAMPUS_CODE );
+
+/**
+ * Makes a get request to the API using the endpoint specified
+ * @param endpoint destination in the API for the request
+ * @param config [optional] additional parameters to send with the request
+ * @returns A thenable promise to use
+ */
```

Comment:
Perhaps able to condense by making line 5 a template literal and then scooting it to the top of the function that adds the endpoint to it, for clarities sake